File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        vector<int> store;

        for (int i = 0; i < nums.size(); i++) {

            bool exists = false;

            for (int j = 0; j < store.size(); j++) {
                if (nums[i] == store[j]) {
                    exists = true;
                    break;
                }
            }

            if (exists) {
                return true;
            }

            store.push_back(nums[i]);
        }

        return false;        
    }
};

File: Data Structures & Algorithms/check-if-array-is-sorted-and-rotated/submission-1.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        rotate = 0
        for i in range(len(nums)-1):
            if nums[i+1] < nums[i]:
                rotate += 1
        if rotate > 1:
            return False
        elif rotate == 1:
            if nums[len(nums)-1] <= nums[0]:
                return True
            else:
                return False
        else:
            return True
